ui/packages/consul-ui/app/components/code-editor/layout.scss

```diff
@@ -32,3 +33,121 @@
 %code-editor > pre {
   display: none;
 }
+
+%code-editor .ember-basic-dropdown-trigger {
+  background-color: var(--tone-gray-900);
+  color: var(--black);
```

Comment:
Could you use `--tone-gray-000` for this please? (instead of --black)

All of the `--tone`s are themeable `--tone-gray-000` is black (in light mode) and `--tone-gray-999` is white. We have this "quick dark mode" thing which just reverses the gray tones (so 000 = white and 999 = black), which gets us almost to dark mode or high contrast mode really easily.

So glad you did it like this though (but I'd like you to change) as it means I know I need to add this to our color docs 👍

Reply:
I must be using the dark mode because 
![image](https://user-images.githubusercontent.com/29378614/140380194-3052e791-67e6-4e3a-9b8d-109f28ef4a7a.png)
comes from this:
![image](https://user-images.githubusercontent.com/29378614/140380227-63d469d7-d6a4-4a08-8b18-278301e1af43.png)
I just pushed this specific change.

Reply:
Oh sorry wrote that wrong 🤦  its the other way around. Have a look here https://github.com/hashicorp/consul/blob/main/ui/packages/consul-ui/app/styles/base/color/ui/themes/light.scss#L1